hieradata/common.yaml

```diff
@@ -351,6 +351,9 @@ govuk::apps::publishing_api::db::backend_ip_range: "%{hiera('environment_ip_pref
 govuk::apps::rummager::nagios_memory_warning: 4600
 govuk::apps::rummager::nagios_memory_critical: 4900
 
+govuk::apps::service_manual_publisher::http_username: "%{hiera('http_username')}"
+govuk::apps::service_manual_publisher::http_password: "%{hiera('http_password')}"
+
```

Comment:
My mistake, I though these were going to be defined in separate hieradata files for integration/staging/prod. In this case since you're referencing things in the same file it makes sense.